README.md

```diff
@@ -97,6 +97,99 @@ In cases where this default variable wrapping can cause problems (e.g. a variabl
 
   `{{#wrap}}[{{nowrap link.text}}]({{nowrap link.url}}){{/wrap}}`
 
+#### Handlebars helpers
+
+In addition to the standard helpers which are part of Handlebars, there are a number of additional helpers defined to assist with Ricardian contract specific issues.
+
+___Since 0.0___
+* `wrap` - Takes no parameters. Block level helper to wrap the contents with `<div>` tags for highlighting. See `Variables` section above for more details.
+
+* `nowrap` - Takes a variable. Indicates that the specified variable *should not* be wrapped with `<div>` tags. See `Variables` section above for more details.
+
+* `symbol_to_symbol_code` - Given a variable containing a 'symbol' string, extract and return only the symbol code.
+
+  Example:
+  ```
+  Given:
+  symbol = '4,EOS'
+  
+  {{ symbol_to_symbol_code symbol }} --> 'EOS'.
+
+* `asset_to_symbol_code` - Given a variable containing an 'asset' string, extract and return only the symbol code.
+
+  Example:
+  ```
+  Given:
+  asset = '2.001 EOS'
+  
+  {{ asset_to_symbol_code asset }} --> 'EOS'.
+
+___Since 0.1___
```

Comment:
Are these references to versions? If so, maybe `__Since v0.1___` would be more clear here